src/renderer/components/Settings/Notifications.tsx

```diff
@@ -13,6 +20,7 @@ export default function Notifications({ desktopSettings }: Props) {
 
   return (
     <>
+      <SettingsHeading>{tx('pref_for_all_accounts')}</SettingsHeading>
```

Comment:
Some ideas here:

1. How about changing the order, making "current account" first and "all accounts" second
2. Remove "current account" heading, as similar to all other settings we can assume that they are mainly for the current account already (except when they are explicitly named to affect the application on a system-level)
3. Disable the "current account" switch as soon as the system-wide setting got turned off

Reply:
<img width="407" alt="Bildschirmfoto 2024-02-13 um 16 19 47" src="https://github.com/deltachat/deltachat-desktop/assets/18725968/7b0b45ad-bbca-448d-85dc-5be3576c0efa">

1. not sure, if we have 3. it makes more sense to have the global setting on top
2. we can only do that if we do 1. and we introduce a new string for the toggle switch for muting the account, otherwise the ui gets confusing.
3. you can still change it in the account sidebar and it still affects the badge color, so I think really disabling the input is more effort (if it should be consistent with the rest of the app, otherwise we could just ignore that you can change it somewhere else.).

Also there will be a new (probably account specific) setting in the future: setting a different/custom notification sound #3583, so keep that in mind when we talk about structuring the dialog. (I think it will become a select that opens a dialog, but still an extra setting that takes up space.